VRCFaceTracking/Params/Expressions/UnifiedExpressions.cs

```diff
@@ -0,0 +1,235 @@
+        MouthPressLeft, // Squeezes the left side lips together vertically and flattens them. Basis on the complex tightening action of the orbicularis oris muscle.
+        MouthTightenerRight, // Squeezes the right side lips together horizontally and thickens them vertically slightly. Basis on the complex tightening action of the orbicularis oris muscle.
+        MouthTightenerLeft, // Squeezes the right side lips together horizontally and thickens them vertically slightly. Basis on the complex tightening action of the orbicularis oris muscle.
+
+        #endregion
+
+        #region Tongue Expressions
+
+        TongueOut, // Combined LongStep1 and LongStep2 into one shape, as it can be emulated in-animation
+
+        // Based on SRanipal tracking standard's tongue tracking.
+        TongueUp, // Tongue points in an upward direction.
+        TongueDown, // Tongue points in an downward direction.
+        TongueRight, // Tongue points in an rightward direction.
+        TongueLeft, // Tongue points in an leftward direction.
+
+        // Based on https://www.naun.org/main/NAUN/computers/2018/a042007-060.pdf
+        TongueRoll, // Rolls up the sides of the tongue into a 'hotdog bun' shape.
+        TongueBendDown, // Pushes tip of the tongue below the rest of the tongue in an arch.
+        TongueCurlUp, // Pushes tip of the tongue above the rest of the tongue in an arch.
+        TongueSquish, // Tongue becomes thinner width-wise and slightly thicker height-wise.
+        TongueFlat, // Tongue becomes thicker width-wise and slightly thinner height-wise.
+
+        TongueTwistRight, // Tongue tip rotates clockwise from POV with the rest of the tongue following gradually.
+        TongueTwistLeft, // Tongue tip rotates counter-clockwise from POV with the rest of the tongue following gradually.
+
+        #endregion
+
+        #region Throat/Neck Expressions
+
+        SoftPalateClose, // Visibly lowers the back of the throat (soft palate) inside the mouth to close off the throat.
+        ThroatSwallow, // Visibly causes the Adam's apple to pull upward into the throat as if swallowing.
+        
+        NeckFlexRight, // Flexes the Right side of the neck and face (causes the right corner of the face to stretch towards.)
+        NeckFlexLeft, // Flexes the left side of the neck and face (causes the left corner of the face to stretch towards.)
+
+        #endregion
+
+        Max
+    }
+
+    /// <summary> 
+    /// Represents the type of Legacy Shape data being sent by UnifiedExpressionData, in the form of enumerated SRanipal shapes.
+    /// </summary>
+    /// <remarks>
+    /// This enum is not intended to be used directly by modules in the final iteration, and instead will be used as a compatibility layer for making the new Unified system backwards compatible with older VRCFT avatars.
+    /// </remarks>
+    internal enum SRanipal_LipShape_v2
```

Comment:
This is currently labelled as internal (a change I did near the start of this PR), but we could re-expose it and make sure that it's named the same as before in VRCFT with the obsolete attr.

Reply:
IMO we shouldn't be redefining classes at all. If its not something we use in the VRCFT namespace, remove it. If we do use it in VRCFT, we should remove all other instances of it from included external tracking modules

Reply:
Currently this enum is only used by the SRanipal conversion and the obsoleted LipMerger combined parameters, otherwise it's not used outside of that and this one shouldn't be used by third-party modules, keeping it internal is probably best. Was just wondering how you wanted to go about exposing VRCFT stuff 😎